### ISSUE TITLE
`<Calendar/>` - (a11y) - Days should have focus-ring ONLY when focused by keyboard

### ISSUE THREAD

**Issue by hepiyellow** [wix/wix-style-react#2780]

**Comment by shlomitc** [comment]
won't be fixed
